### ISSUE TITLE
WurstPlusTwo and Three is Token,IP,PhoneNumber,Email... Grabber NOT INSTALL

### ISSUE THREAD

**Issue by PlanetaCZ** [TrvsF/wurstplus-two#96]
**### Shit client**

**Comment by ampdoescode** [comment]
Huh?? Prove it.

**Comment by TrvsF** [comment]
wurst+3??!?!?! this is my new favourite issue ty x

**Comment by maxgamesNL** [comment]
This is totally true, it also makes your computer eat you in your sleep

**Comment by phobologicalt** [comment]
oMg I nEvEr KnEw I wIlL uNiNsTaLl NoW

**Comment by UttamOnTop** [comment]
Lmao cope

**Comment by UttamOnTop** [comment]
hes just mad that this client has good ca and auto32k

**Comment by villkaa-2b2t** [comment]
is the backdoor wurst+2 public?

**Comment by TrvsF** [comment]
https://github.com/TrvsF/discord-token-logger

**Comment by villkaa-2b2t** [comment]
ok so that's the token logger one and this one is not correct or not?

**Comment by maxgamesNL** [comment]
> ok so that's the token logger one and this one is not correct or not?

Yes, this one is safe and those files are just an example

**Comment by Mskeen000** [comment]
Omg TrvsF have my porn stash, why dont you tell me before i download it????!?

**Comment by wijiler** [comment]
No no it isnt

**Comment by maxgamesNL** [comment]
> No no it isnt

We know, this whole tread is a big meme

**Comment by wijiler** [comment]
> > No no it isnt
> 
> We know, this whole tread is a big meme

yes yes I know

**Comment by Mskeen000** [comment]
I forgot that Github is sending messages to my email, and someone send me a message and i thought that TrvsF get my email from W+2 😆. 3 seconds later i realised that it was github....

**Comment by DankDupe** [comment]
god that computer eating you in your sleep made me fall of my chair xD

**Comment by maxgamesNL** [comment]
> god that computer eating you in your sleep made me fall of my chair xD

Thx

**Comment by DankDupe** [comment]
is this safe?

**Comment by maxgamesNL** [comment]
> is this safe?

This client(wurst+2) is totally safe. That other thing(the token logger) is just an example on how something like that would look.

I myself use this client and for as far as I know it is safe.

**Comment by Alexrandomacc** [comment]
How do i download it cause i downloaded the code but theres not java thing that i put in my mc folder can someone help me

**Comment by maxgamesNL** [comment]
> How do i download it cause i downloaded the code but theres not java thing that i put in my mc folder can someone help me 

If you are on pc website then it should be at the right of your screen

**Comment by Aholicknight** [comment]
show proof that it is lmao

**Comment by phobologicalt** [comment]
> show proof that it is lmao

dOnT yOu KnOw ThAt TrAvIs LoGgEd CrYsTaLiNqQ fItMc AnD wOlFiEs SaId11!11!11!11!!111

**Comment by Aliihsann365** [comment]
Wurst+2 İs Safe?

**Comment by phobologicalt** [comment]
> Wurst+2 İs Safe?

yes, this version is 100% safe, but only get it from here. If you download it off of discord or youtube its probably not safe

**Comment by Aliihsann365** [comment]
> > Wurst+2 İs Safe?
> 
> yes, this version is 100% safe, but only get it from here. If you download it off of discord or youtube its probably not safe

Thanks

**Comment by Nutelaspaceman** [comment]
> 
> 
> **### Shit client** **TOKEN LOGGER**

drunk guy have u even looked in the file by yourself?
